test/fixtures/es-module-loaders/builtin-named-exports-loader.mjs

```diff
@@ -28,7 +28,7 @@ export function load(url, context, next) {
       source: generateBuiltinModule(urlObj.pathname),
       format: 'commonjs',
     };
-  } else if (context.format === 'commonjs') {
+  } else if (context.format === undefined || context.format === null || context.format === 'commonjs') {
```

Comment:
nit
```suggestion
  } else if (context.format == null || context.format === 'commonjs') {
```

Reply:
No thanks, I prefer to be explicit. I want it to be clear that we're checking for these values.

Many users, myself included (at work), have lint rules prohibiting `==` under any circumstances. It looks wrong and it's not as type-safe.